content/talks/resilient-systems.rst

```diff
@@ -0,0 +1,117 @@
+public: yes
+speakers: [miriam]
+headline:
+  - type: Training & Consulting
+    tagline: |
+      Upgrade your app with resilient systems training, tooling & consulting
+image:
+  - src: 'talks/miriam.jpg'
+event_type: workshop
+grid_template: true
+brag: |
+  Upgrade your app with resilient systems
+  training, tooling & consulting
+modules:
+  - title: Resilient & Universal Code
+    summary: |
+      Browser & device support
+      doesn't have to be difficult and expensive.
```

Comment:
I don't actually know whether "don't" or "doesn't" is correct here. Both sound kind of wrong to me. :) Could just leave it, or re-word to avoid.

Reply:
I was able to accept your other change from my phone :) want to suggest something here? :p